src/container/BooksForm.js

```diff
@@ -0,0 +1,52 @@
+/* eslint-disable react/no-unused-state, class-methods-use-this, no-unused-vars */
+
+import React, { Component } from 'react';
+
+class BooksForm extends Component {
+  constructor(props) {
+    super(props);
+
+    this.state = {
+      title: '',
+      category: '',
+    };
+
+    this.handleChange = this.handleChange.bind(this);
+  }
+
+  handleChange(event) {
+    // e.preventDefault();
+    // const { name, value } = event.target;
+    // switch (name) {
+    //   case 'title':
+    //   case 'category':
+    //     this.setState(prevState => ({ ...prevState, [name]: [value] }));
+    //     break;
+    //   default:
+    //     this.setState({ [name]: value });
+    // }
+  }
```

Comment:
Keep in mind to always remove commented code in your codebase before requesting a CRR.
It’s better to delete the commented code. If you need to review a previous version of your code, use git history.